packages/cosmwasm-schema-cli/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "@satlayer/cosmwasm-schema-cli",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Generate CosmWasm schemas from Rust code using quicktype.",
+  "bin": {
+    "schema-gen": "./schema-gen.mjs"
+  },
```

```suggestion
  "repository": {
    "url": "git+https://github.com/satlayer/satlayer-bvs"
  },
  "license": "MIT",
  "bin": {
    "schema-gen": "./schema-gen.mjs"
  },
  "files": [
    "schema-gen.mjs"
  ],
```

Some things are missing, `files` should be added, otherwise it might not be included (not sure if they pick up bin automatically).
The repository field is required for provenance generation.